validator/client/validator.go

```diff
@@ -599,3 +601,17 @@ func (v *validator) logDuties(slot uint64, duties []*ethpb.DutiesResponse_Duty)
 func validatorSubscribeKey(slot uint64, committeeID uint64) [64]byte {
 	return bytesutil.ToBytes64(append(bytesutil.Bytes32(slot), bytesutil.Bytes32(committeeID)...))
 }
+
+//This will keep account of all pubkeys' attestations.  The total variables other than totalDistance include missed attestations.
+type aggregateStats struct {
+	startEpoch       uint64
+	includedAttests  uint64
+	totalAttestSlots uint64
+	totalDistance    uint64
```

Comment:
```suggestion
	totalDistance    uint64
```
`totalDistance` need more clarity. I'm opting to leave these status out since it's not really useful from bird eye view

Reply:
totalDistance allows for the average inclusion distance stat/calculation (total inclusion distance / number of included attestations).  Do you think we should remove this?